applications/FluidDynamicsApplication/custom_elements/fluid_element.h

```diff
@@ -92,7 +92,11 @@ class FluidElement : public Element
     typedef PointerVectorSet<Dof<double>, IndexedObject> DofsArrayType;
 
     /// Type for shape function values container
-    typedef MatrixRow< Matrix > ShapeFunctionsType;
+    #ifdef KRATOS_USE_AMATRIX
```

Comment:
this should not be in this PR, I think you need to merge master first

Reply:
idk why the kraken has included this commit :S

Reply:
> idk why the kraken has included this commit :S

That's what happens when you awaken the Kraken :laughing: